notary/content.md

```diff
@@ -0,0 +1,82 @@
+%%LOGO%%
```

Comment:
The script will replace it with an empty line but that would produce invalid markdown according to markdownfmt so it would cause the job to fail.